Documentation/ApiOverview/Deprecation/Index.rst

```diff
@@ -0,0 +1,82 @@
+.. include:: ../../Includes.txt
+
+.. _deprecation:
+
+===========
+Deprecation
+===========
+
+Since TYPO3 4.3, calls to deprecated functions are logged to track usage of
+deprecated/outdated methods in the TYPO3 Core. Developers have to make sure to adjust their code to avoid
+using this old functionality since deprecated methods will be removed in future TYPO3 releases.
+
+.. _deprecation_introduction:
+
+Introduction
+============
+
+Deprecations since TYPO3 9 use the PHP method :php:`trigger_error('a message', E_USER_DEPRECATED)` and run
```

Comment:
Should we mention here that the related `GeneralUtility` methods have been deprecated in this version and should not be used anymore?

Reply:
this is a straight backport from the info that is currently available in master. I'm not sure it's worth the effort at this point to add essentially outdated info. If you think it helps people we could do it, of course.

Reply:
Yes, it's always useful to have a hint at a specific location. I think it's effort worth spend and added this now.